app/views/spree/api/affirm/payload.json.jbuilder

```diff
@@ -0,0 +1,77 @@
+json.config do
```

Comment:
would be the same case for AMS, (I do love AMS though :) ). I like the PORO approach